classes/Model/AnnoucementManagement.php

```diff
@@ -7,6 +7,48 @@
 
 namespace MySociety\TheyWorkForYou\Model;
 
+
+
+function is_valid_item($item) {
+    // used in an array sort to filter out invalid items
+
+    // update this when merging with the language aware branch
+    $current_language = "en";
+
+    // set default language
+    if (isset($item->lang)){
+        $language = $item->lang;
+    } else {
+        $language = 'en';
+    }
+
+    // set default for published
+    if (isset($item->published)){
+        $published = $item->published;
+    } else {
+        $published = true;
+    }
+
+    // set default start_time (in the past)
+    if (isset($item->start_time)){
+        $start_time = $item->start_time;
+    } else {
+        $start_time = "2000-01-01T00:00:00";
+    }
+
+    // set default end_time (in the future)
+    if (isset($item->end_time)){
+        $end_time = $item->end_time;
+    } else {
+        $end_time = "2100-01-01T00:00:00";
+    }
+
+    return $published
+        && $language == 'en'
```

Comment:
I realise you say will need updating when merged with language branch, but this line should be using $current_language already.